Source/Fuse.Reactive.Expressions/VarArgFunction.uno

```diff
@@ -10,7 +10,7 @@ namespace Fuse.Reactive
 	public abstract class VarArgFunction: Expression
 	{
 		/** Holds information about an argument to a `VarArgFunction`. */
-		protected class Argument 
+		internal class Argument 
```

Comment:
This needs to be `internal protected` -- removing the `protected` is the cause of several of the test errors.